test/connectors/httpParsers/EisHttpParsersSpec.scala

```diff
@@ -82,22 +82,25 @@ class EisHttpParsersSpec extends SpecBase {
       val response = HttpResponse(Status.OK, body, headers)
 
       "return an error" in {
-        createCaseHttpParser.read("", "", response) mustBe Left(ErrorModel(Status.OK, "INVALID JSON"))
+        createCaseHttpParser.read("", "", response) mustBe Left(EisError.UnexpectedError(Status.OK, "Received invalid JSON"))
       }
 
     }
   }
 
-  "Parsing a non-200 response" should {
+  "Parsing a 400 response" should {
 
     val headers: Map[String, Seq[String]] = Map("x-correlation-id" -> Seq(correlationId))
-    val body: JsObject = Json.obj()
-    val response = HttpResponse(Status.BAD_REQUEST, body, headers)
 
-    "return an error" in {
-      createCaseHttpParser.read("", "", response) mustBe Left(ErrorModel(Status.BAD_REQUEST, "Non-success response"))
+    "the response is a specific EIS error" in {
```

Comment:
can we have a test for the 04 error message too please. And maybe 06 just to show that parses to InternalServer error

Reply:
this is already covered in the `UpdateCaseErrorSpec`